### DISCUSSION TITLE
open.openApp and fkill not working

### DISCUSSION THREAD

**Discussion by AmodeusR** [johnlindquist/kit#1182] [Q&A]
Hello, I'm trying to use these commands but they don't seem to have an effect. Here's how I wrote the code:

```js
// Name: Open Notion

import "@johnlindquist/kit";

const open = await npm("open");
const fkill = await npm("fkill");

const app = "C:/Program Files/Notion Enhanced/Notion Enhanced.exe";

try {
  console.log("Opening")
  await open.openApp(app);
  await wait(2000);
  console.log("Closing")
  fkill(app, { silent: true });
  await wait(1000);
  console.log("Reopening")
  await open.openApp(app);
} catch (error) {
  console.log("Something went wrong");
}
```
I made this code using as reference [this one](https://github.com/johnlindquist/kit/discussions/917). When I try executing the code nothing happens besides the logs and waits.
I tried using just the program name and the full path but both don't have any effect. If I try to execute just `open(path)` it works, though.

**Comment by johnlindquist** [comment]
@AmodeusR Yeah, just use `open`.

`open` is even included as a global now, so a script that simply says:

```js
open("cmd.exe")
```

Should be enough

**Reply by AmodeusR** [reply]
What about the closing with fkill? It also doesn't work.
